### ISSUE TITLE
Implement YOLOv1 training script on FSOCO

### ISSUE THREAD

**Issue by carlostojal** [carlostojal/obj_detection_playground#10]
Implement the script to train YOLOv1 on the FSOCO dataset.

- [x] Implement bounding box conversion utilities.
- [x] Implement the loss function.
- [x] Implement the dataset class.
- [x] Split the dataset in the FiftyOne dataset creation.
- [x] Implement the training script.

**Comment by carlostojal** [comment]
Closed by #14
